README.md

```diff
@@ -1,5 +1,9 @@
 ![](assets/banner.png)
 
+# Fork with fresh TypeScript (5.1.6) version inside
+
+Original readme
+
```

Comment:
Let's remove this from the PR, you're welcome to leave this in your fork though!

Reply:
Done! Sorry, missed notification from gh